pkg/doc/util/jwkkid/kid_creator_test.go

```diff
@@ -32,9 +36,23 @@ func TestCreateKID(t *testing.T) {
 	require.NoError(t, err)
 	require.NotEmpty(t, kid)
 
+	// now try building go-jose thumbprint and compare its base64URL with kid above
+	// they should not match since go-jose's thumbprint is built from a wrong Ed25519 JWK.
+	jwk, err := jose.JWKFromPublicKey(pubKey)
+	require.NoError(t, err)
+
+	goJoseTP, err := jwk.Thumbprint(crypto.SHA256)
+	require.NoError(t, err)
+
+	goJoseKID := base64.RawURLEncoding.EncodeToString(goJoseTP)
+	require.NotEqual(t, kid, goJoseKID)
+
+	// now build bad thumbprint and ensure it matches goJoseTP
```

Comment:
Where is a test showing we are building the correct kid

Reply:
according to the pending pr, the fix is here https://github.com/square/go-jose/pull/324/files#diff-adb413cfea72e48e90449f9604b1472d63b22160f9c1230bb22517ece6a0fa89R334

In AFG, we build a correct JWK marshalled string then do the same as go-jose. I have not searched if there's an official source of JWK Thumbprint. Unfortunately IETF [RFC 7638](https://tools.ietf.org/html/rfc7638) has no mention of JWK's of Ed25519 keys and contains only an RSA example, so I take it go-jose's format is valid.

Reply:
We need a test that demonstrates the correct behavior, not just that it isn't the wrong one.

Reply:
added `TestCreateKIDFromFixedKey` further down at line 122

Reply:
* JWK thumbprint algo: https://tools.ietf.org/html/rfc7638#section-3
* JWK of public Ed25519 key: https://tools.ietf.org/html/rfc8037#appendix-A.2

Reply:
I'll update the positive KID  test with https://tools.ietf.org/html/rfc8037#appendix-A.2 @llorllale , thanks

Reply:
updated, Ed25519 JWK Thumbprint test now matches [rfc 8037 A.3](https://tools.ietf.org/html/rfc8037#appendix-A.3) example